tests/testing_config/custom_components/test/sensor.py

```diff
@@ -20,6 +20,10 @@
     sensor.DEVICE_CLASS_TIMESTAMP: "hh:mm:ss",  # timestamp (ISO8601)
     sensor.DEVICE_CLASS_PRESSURE: "hPa",  # pressure (hPa/mbar)
     sensor.DEVICE_CLASS_POWER: "kW",  # power (W/kW)
+    sensor.DEVICE_CLASS_CURRENT: "A",  # current (A)
+    sensor.DEVICE_CLASS_ENERGY: "kWh",  # energy (Wh/kWh)
+    sensor.DEVICE_CLASS_POWER_FACTOR: "None",  # power factor (no unit, min: -1.0, max: 1.0)
```

Comment:
`None` should be `NoneType` not a string.

Reply:
fixed, sorry for the misunderstanding